### ISSUE TITLE
Does it keyboard aware?

### ISSUE THREAD

**Issue by naytun** [joseestrella89/react-native-dialog-input#16]
When keyboard shows up it needs to push up so that keyboard will not cover the buttons.

**Comment by rodriguezmanu** [comment]
+1

**Comment by gonzalovinas** [comment]
Under a similar post title I would say that Android emulator does not capture the physical keyboard, although emulated keyboard works just fine!!

**Comment by fpena** [comment]
I solved this temporarily disabling autoCorrect. 
```
textInputProps={{ autoCorrect: false }}
```

**Comment by Senocico** [comment]
Also you can fix this like this: 

`dialogStyle={{marginBottom: 100}}`

**Comment by ikismail** [comment]
Any fix for this.
I'm too facing this issue. Cant able to control the Buttons of the dialog input when the keyboard popup

![image](https://user-images.githubusercontent.com/17190913/71326278-b094a680-251e-11ea-9bb0-29515c73b0d7.png)

**Comment by akiladevinda** [comment]
@ikismail  have you figure this out? I'm having the same problem only on iOS

**Comment by WQvist** [comment]
Any solution on iOS?

**Comment by elmcapp** [comment]
> textInputProps={{ autoCorrect: false }}

Yes this fixes the issue however it has to be a bug because now text is not auto correct.

**Comment by the-smart-home-maker** [comment]
As @Senocico stated - this works for me: `dialogStyle={{marginBottom: 100}}` - not perfect and probably not working for all screen sizes but for the most situations it solves the problem
